src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1549,7 +1549,7 @@ public function getLanguages()
         foreach (array_keys($languages) as $lang) {
```

Comment:
May I suggest we remove all these foreach+array_keys also? They just lower perf and memory usage